utils/utils.cfg

```diff
@@ -1289,13 +1289,26 @@ $item_info.description"
                                 to_variable=beel_store
                             [/unit]
                         )}
-                        {VARIABLE beel_store_level_add {POWER}}
-                        {VARIABLE_OP beel_store_level_add multiply 5}
-                        {VARIABLE_OP beel_store.level add $beel_store_level_add}
```

Comment:
Please keep this for compatibility with 1.16.

Reply:
apply_to=level works on 1.16 and 1.17 on the same way. The first with the new lua code, and the second natively.

Reply:
Ahaa.... okay.